### PR TITLE
JP-638 Fixed issue with handling schedule updates

### PR DESCRIPTION
**Pull request by pies**

### CODE REVIEW

File: src/cli/DownloadCommand.ts [hunk 1]
```diff
@@ -30,14 +30,14 @@ export class DownloadCommand implements CLICommand {
...
 
-    try {
-      await Promise.all(
-        files.map(f => this.sftp.fastGet(this.directory + f, outputDirectory + f))
-      );
-    }
-    catch (err) {
-      console.error(err);
-    }
+    // try {
```
Comment (orzeuek) [inline src/cli/DownloadCommand.ts]: can we remove it ?

**Review by orzeuek — APPROVED** [review]
Gentle, elegant solution!

I love the idea about "remove orphan" :) 
Maybe we should implement other methods, like "remove AIDS", "remove global hunger", "remove cancer" and world would become a better place :D

File: src/cli/DownloadCommand.ts (hunk 1 above)
Comment (pies) [reply src/cli/DownloadCommand.ts]: Sorry, that's my debug code.

**Comment by pies** [comment]
> I love the idea about "remove orphan" :)
> Maybe we should implement other methods, like "remove AIDS", "remove global hunger", "remove cancer" and world would become a better place :D

I would not tweet that thought :)

**Pull request merged**